source-repo-scripts/source_changelog.bash

```diff
@@ -1,23 +1,19 @@
 #!/bin/bash
 # bash source_changelog.bash <PREV_VER>
 
-PREV_TAG=$1
+PREV_VER=$1
 
 git fetch --tags
 
-REPO=$(basename `git rev-parse --show-toplevel`)
-REPO="${REPO/ign-/gz-}"
-MAJOR=${PREV_TAG%.*.*}
-BRANCH=${REPO/sdformat/sdf}${MAJOR}
-TAG="${REPO}${MAJOR}"
+REPO=$(basename $(git remote get-url origin))
 
-# TODO(chapulina) Support Garden tags and branches, which will start with gz-
-TAG="${TAG/gz-/ignition-}"
-TAG="${TAG/sim/gazebo}"
-BRANCH="${BRANCH/gz-/ign-}"
-BRANCH="${BRANCH/sim/gazebo}"
+# Find tag that ends with _$PREV_VER
+PREV_TAG=$(git tag | grep "_${PREV_VER}$")
 
-COMMITS=$(git log ${BRANCH}...${TAG}_${PREV_TAG}  --pretty=format:"%h")
+# Compare current branch to PREV_TAG
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+COMMITS=$(git log ${BRANCH}...${PREV_TAG}  --pretty=format:"%h")
```

Comment:
Not sure what have changed here but the retrieval of the PR number is now broken:
```
+ for COMMIT in $COMMITS
++ git log --format=%s -n 1 862464c
+ TITLE_FULL='Close branch default_bump_to_v3'
+ TITLE='Close branch default_bump_to_v3'
+ PR='Close branch default_bump_to_v3'
+ PR='Close branch default_bump_to_v3'
+ echo '1. Close branch default_bump_to_v3'
1. Close branch default_bump_to_v3
+ echo '    * [Pull request #Close branch default_bump_to_v3](https://github.com/gazebosim/ign-cmake.git/pull/Close branch default_bump_to_v3)'
    * [Pull request #Close branch default_bump_to_v3](https://github.com/gazebosim/ign-cmake.git/pull/Close branch default_bump_to_v3)
+ echo ''
```

Reply:
The script doesn't handle commits that weren't squash-merged from GitHub pull requests. In this case, those are bitbucket hg commits that haven't yet been released (wow). Currently the script user just filters those out. Some future work could be trying to do more of this automatically.

Reply:
wow! thanks for clarifying